packages/fiori/src/ShellBar.ts

```diff
@@ -578,12 +762,29 @@ class ShellBar extends UI5Element {
 		});
 
 		this._observeMenuItems();
+		this._observeAdditionalContextItems();
```

Comment:
probably we need to add some caching whether item is observer and when not otherwise this is constantly executing

Reply:
we also need to disconnect observers once the DOM is removed from the dom. Probably it should be attached / detatched where resize handler is doing it.